src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -38,17 +41,40 @@ DEFINE_string(session_name,
               "session_name: The session name (ClusterID) of the cluster.");
 DEFINE_string(ray_commit, "", "The commit hash of Ray.");
 
+namespace {
+// GCS server output filename.
+constexpr std::string_view kGcsServerLog = "gcs_server.out";
```

Comment:
get_log_file_handles can create names like `gcs_server.2.out` if `gcs_server.out` and `gcs_server.1.out` both exists. Do we have such thing in spdlog?

Reply:
All rotated logs are suffixed with id, similar to what you're described.

Reply:
I meant already-exists file detection and renaming. Discussed offline, we need to keep that in python and pass into cpp as arg of log_file.

Reply:
Discussed offline, the log sink filename is determined at python side and passed to C++ side. Mark as resolved.